VMEncryption/test/test_handler_util.py

```diff
@@ -33,26 +33,34 @@ def setUp(self):
         self.hutil = HandlerUtil.HandlerUtility(self.logger.log, self.logger.error, "AzureDiskEncryptionForLinux")
         self.hutil.patching = self.distro_patcher
         # invoke unit test from within main for setup (to avoid having to change dependencies)
-        # then move cwd to parent to emulate calling convention of guest agent 
-        if os.getcwd().endswith('main'):
-            os.chdir(os.path.dirname(os.getcwd()))
-        else:
-            self.logger.log(os.getcwd())
+        handlerPath = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
consider replacing with mock, or add steps to chown /var/lib and /var/log folders